libs/wallet/src/vega-transaction-dialog/vega-transaction-dialog.tsx

```diff
@@ -5,18 +5,34 @@ import {
   addDecimalsFormatNumber,
   t,
 } from '@vegaprotocol/react-helpers';
-import type { VegaTxState } from '@vegaprotocol/wallet';
-import { VegaTxStatus } from '@vegaprotocol/wallet';
-import type { OrderEvent_busEvents_event_Order } from '../hooks/__generated__/OrderEvent';
+import type { VegaTxState } from '../use-vega-transaction';
+import { VegaTxStatus } from '../use-vega-transaction';
+
+export interface Market {
+  name: string;
+  positionDecimalPlaces?: number;
+  decimalPlaces: number;
+}
+
+export interface Order {
+  status: string;
+  rejectionReason: string | null;
+  size: string;
+  price: string;
+  market: Market | null;
+  type: string | null;
+}
 
 interface OrderDialogProps {
   transaction: VegaTxState;
-  finalizedOrder: OrderEvent_busEvents_event_Order | null;
+  finalizedOrder: Order | null;
+  title?: string;
 }
 
-export const OrderDialog = ({
+export const VegaTransactionDialog = ({
```

Comment:
chore (blocking): not all transactions are orders, so this should probably be VegaOrderTransactionDialog

Reply:
updated in last commit